types/react-native/index.d.ts

```diff
@@ -4399,6 +4399,10 @@ export interface ModalProperties {
      * @platform ios
      */
     onDismiss?: () => void;
+    /**
+     * The `presentationStyle` determines the style of modal to show
```

Comment:
Can you add a `@platform ios` comment like the above ones have?

Reply:
My bad - added it now.

Reply:
Thanks!